docs/index.md

```diff
@@ -1,639 +1,129 @@
-subshells with dependencies provided and configured for you.
-Even better, these environments layer so you can prepare different environments
-for different contexts and stack them when needing to work across contexts.
+# What is Flox?
 
-Finally, Flox environments are portable across architecture,
-operating system, and the entire software lifecycle.
+Flox is a next-generation, language-agnostic package and environment manager.   
 
-Let's see what it looks like to bring Flox to one of your existing projects.
+  :material-arrow-right-thick: Define everything your environment needs—packages, tools, environment variables, and services—in one manifest.  
+  :material-arrow-right-thick: Switch between environments easily, share them with your team, and keep everything in version control.   
+  :material-arrow-right-thick: Use the same setup across macOS and Linux, on both x86 and ARM.     
 
-## Quick start
 
-1. Install Flox.
-2. Create an environment with `flox init`.
-3. Find your dependencies with `flox search` and `flox show`.
-4. Install dependencies with `flox install`
-5. Add environment variables, shell hooks, and services with `flox edit`.
-6. Enter the environment with `flox activate`
-7. Play around, see if everything works, what's missing, etc.
-8. Leave the environment via `exit` or Ctrl-D.
+Flox achieves isolation through pre-configured sub-shells, not containers, so it works seamlessly with your existing tools, shells, and dotfiles. Under the hood, Flox uses Nix to ensure reproducibility—without requiring you to learn Nix.    
 
-Let's walk through those steps and explain in more detail.
+Flox makes it easy to work locally, test in CI, and deploy to production—all with the same environment.
 
-## Install Flox
 
-Start by [installing `flox`][install_flox].
+ <iframe width="550" height="300" src="https://www.youtube.com/embed/aidi5svDml8?si=rrgQ6a0oQzdFNgWs" title="What is Flox?" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe> 
 
-## Create an environment
 
-We'll start by creating an environment for your project,
-and we'll pretend that your project is called `my_project`.
 
-Flox installs packages into _environments_ as opposed to installing them into
-globally accessible directories in various places on your system.
-This allows you to have different versions of your tools across different
-projects without them interfering or conflicting.
+## Get Started
+<div class="grid cards" markdown>
 
-`cd` into your project and initialize the environment with [`flox init`][init].
+-   :octicons-terminal-24:{ .lg .middle } __Quick start with the Flox CLI__
 
-For Python, Node.js, and Go projects this will display a prompt asking if you
-would like Flox to automatically do some language-specific setup for you.
+    ---
 
-=== "Python"
+    Install `flox` with `brew` and get your dev 
```

Comment:
```suggestion
    Install `flox` and get your dev 
```

Reply:
How about 
> Install `flox` with `apt`, `yum`, or `brew` and ...

Reply:
Ah, sorry I didn't explain. The reason for calling out brew specifically is because we saw in several research sessions that (1) people looked for brew as an install method but didn't see it on the website so they chose the download instead and (2) when someone did find or use brew, they were delighted to use that method. 

My hypothesis is that if you just put the word `brew` front and center when you mention install, they won't have to click to the Install docs, and can just get started even more quickly. Adding `apt` and `yum` would do the same for those users. 

How about this:
![image](https://github.com/user-attachments/assets/1d654f08-4e7a-41dc-a4c7-86faf60c3c1a)

Reply:
That is a fine compromise.

I understand that we have that research result, but I don't agree that putting brew front and center is the best decision. I weigh those research experiences in my mind lower than the desire not to confuse people about the role Flox plays in their stack.